torch_geometric/nn/conv/cluster_gcn_conv.py

```diff
@@ -69,17 +72,27 @@ def reset_parameters(self):
     def forward(self, x: Tensor, edge_index: Adj) -> Tensor:
         edge_weight: OptTensor = None
         if isinstance(edge_index, Tensor):
+            is_sparse_tensor = is_torch_sparse_tensor(edge_index)
+            if is_sparse_tensor:
+                edge_index, _ = to_edge_index(edge_index)
```

Comment:
```suggestion
                edge_index, _ = to_edge_index(edge_index.t())
```
wouldn't that work as well? Would need to drop the requirement to treat `col, row` differently in the later part of the code.

Reply:
Got it!